JetStreamDriver.js

```diff
@@ -181,47 +181,27 @@ function uiFriendlyDuration(time) {
     return `${time.toFixed(3)} ms`;
 }
 
+// TODO: Cleanup / remove / merge. This is only used for caching loads in the
+// non-browser setting. In the browser we use exclusively `loadCache`, 
+// `loadBlob`, `doLoadBlob`, `prefetchResourcesForBrowser` etc., see below.
 const fileLoader = (function() {
```

Comment:
future-nit: we could actually start using modules instead of these "hacky" IFEEs to hide the classes.